### PR TITLE
[stable/kiam] Use correct server name when generating certificates

### PR DESCRIPTION
**Pull request by DReigada**
Signed-off-by: Daniel Reigada <daniel@codacy.com>

#### What this PR does / why we need it:

The kiam chart generates self-signed certificates for the agent and server. The generated server certificate uses a DNS name for the service, which does not match the service created when the chart is used as a sub chart. This happens because we do not use the full name of the service.

#### Which issue this PR fixes
This might be related to #13970

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by danmx** [comment]
rebase maybe?

**Comment by DReigada** [comment]
@danmx I will gladly rebase it, but it seems rather pointless to keep doing it if this doesn't get any feedback.

**Comment by hobti01** [comment]
We are also looking forward to this fix, it would be great to have it merged.

**Comment by tiagomeireles** [comment]
I am also needing this fix. Can we get this reviewed and merged?

**Comment by mikesplain** [comment]
Looking for this too, maybe we can get some more eyes on it to ensure it doesn't keep getting stale.

/assign ewbankkit

**Comment by maorfr** [comment]
/ok-to-test
/lgtm

sorry this has been waiting for so long

File: stable/kiam/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kiam
-version: 2.4.3
+version: 2.4.4
```
Comment (danmx) [inline stable/kiam/Chart.yaml]: current version in `master` branch is `2.5.0`

**Pull request merged**